### DISCUSSION TITLE
Welcome to sunbird.org-docs Discussions!

### DISCUSSION THREAD

**Discussion by vrayulu** [project-sunbird/sunbird.org-docs#644] [60 General Discussions]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vsowjanyarani** [comment]
Test comments

**Reply by vrayulu** [reply]
test reply

**Reply by joel-ekstep** [reply]
Replying to reply to reply.

**Comment by joel-ekstep** [comment]
Thank you, this is a very useful feature.

**Reply by joel-ekstep** [reply]
Appears to be no upvoting, downvoting possible yet. 
@vrayulu There is tagging, so that's cool.

**Reply by joel-ekstep** [reply]
@vrayulu Are you getting notifications via email (or in your notifications icon in github)?

**Reply by vrayulu** [reply]
@joel-ekstep - by tagging, do you mean "reactions"? i am looking for labels (like what GitHub have for issues), but it is not enabled for discussions yet.

**Reply by vrayulu** [reply]
yes.. i am getting both email and github notifications...

**Reply by vrayulu** [reply]
another missing thing - file attachments..

**Reply by joel-ekstep** [reply]
> @joel-ekstep - by tagging, do you mean "reactions"? i am looking for labels (like what GitHub have for issues), but it is not enabled for discussions yet.

No @vrayulu , I'm referring to being able to bring your attention to it - like how I'm doing in this line.

**Comment by joel-ekstep** [comment]
Looks like if someone replies to replies, then beyond 3 reply-to-replies it collapses under a View More.

**Reply by joel-ekstep** [reply]
Regarding notifications, it's not showing up on my github - but it does come as emails (which I usually route into a different archive folder in my inbox). If most devs also route notifications from github into a different folder, we may have an issue here.

Whats the usual behaviour regarding Github notifications, that devs exhibit, @vrayulu ?

**Reply by vrayulu** [reply]
Developers rely on GitHub email notifications a lot for PR requests and approvals... so, GitHub should work well for discussions also.

**Reply by joel-ekstep** [reply]
@vrayulu Are you able to create new categories in the home page?

P.S. Edits seem possible.

**Reply by vrayulu** [reply]
yes.. i created a new category - "Sunbird Installation and Configuration"...

**Reply by joel-ekstep** [reply]
Tested - We can move chats from category to category.

**Comment by joel-ekstep** [comment]
Doesn't look like we can pin particular posts to the top of a category, though.

**Comment by joel-ekstep** [comment]
Also, categories can't be marked private now.

**Comment by surendrasinghs** [comment]
This seems so much super fun! 🥳

**Reply by vrayulu** [reply]
@joel-ekstep - misusing the admin privilege...

**Comment by vrayulu** [comment]
@joel-ekstep - testing your notification settings

**Comment by joel-ekstep** [comment]
Here's a private conversation that once happened:
https://github.com/orgs/project-sunbird/teams/auth/discussions/3
